ibexa/fieldtype-richtext-rte/4.6/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "aliases": [],
+  "bundles": {
+    "Ibexa\\Bundle\\FieldTypeRichTextRTE\\FieldTypeRichTextRTEBundle": ["all"]
+  },
+  "copy-from-recipe": {
+    "config/": "%CONFIG_DIR%/"
+  },
+  "add-lines": [
+    {
+        "file": "composer.json",
+        "content": "\"ckeditor-rte-copy.sh\": \"script\"",
```

Comment:
Note: https://github.com/ibexa/fieldtype-richtext-rte/blob/main/bin/ckeditor-rte-install.sh script must be executed before we compile FE assets